account_invoice_subscription_per_contact/readme/CONFIGURE.md

```diff
@@ -0,0 +1,3 @@
+#. Go to a partner and subscribe a user (Marc Demo for example) to the subtype 'Invoice Created'.
+#. Go to Invoicing > Customers > Invoices and creates an invoice to the previous contact (or a child contact).
```

Comment:
Explain here about both possibilities to subscribe the children or the parent one.